### ISSUE TITLE
The input field is not focused.

### ISSUE THREAD

**Issue by vzhuqin** [dotnet/machinelearning-modelbuilder#1880]
**System Information (please complete the following information):**
 - Model Builder Version (available in Manage Extensions dialog): 16.9.1.2152802
 - Microsoft Visual Studio Enterprise 2019: 16.11.5

**Describe the bug**
- On which step of the process did you run into an issue: Data page
- Clear description of the problem: Except image classification, the input fields for other scenarios are not focused.

**To Reproduce**
1. Select Create a new project from the Visual Studio 2019 start window;
2. Choose the C# Console App (.NET Core) project template with .Net 5.0;
3. Add model builder by right click on the project;
4. Select one scenario and environment;
5. Navigate to Data page, find that the input field is not focused.

**Expected behavior**
The input field on Data page should be focused for all scenarios.

**Screenshots**
![image](https://user-images.githubusercontent.com/81727020/139399942-b994256d-df92-4f6f-9bd8-9b8e4fb96517.png)

**Additional context**
not happen on Image Classification scenario.
![image](https://user-images.githubusercontent.com/81727020/139400036-33f2b71c-c39f-4c44-a121-069d0f29b037.png)

**Comment by beccamc** [comment]
Regression in the OD data control.

**Comment by vzhuqin** [comment]
@beccamc this issue also happens on Data classification, Value predication, Recommendation scenarios, please take a look again?
Fixed for object detection scenario on main: 16.9.1.2155901
![image.png](https://images.zenhubusercontent.com/60653e492390a08ea616cb10/0df6d78c-a511-422c-9599-f25b0c42fdc2)

**Comment by beccamc** [comment]
The focus for the text scenarios goes to the file/sql radio buttons. Focus is supposed to go to the first element you can interact with.

**Comment by vzhuqin** [comment]
Yes, thanks for your explanation. Focus on the file type for the other three text scenarios.
![image.png](https://images.zenhubusercontent.com/60653e492390a08ea616cb10/f6045c03-b01c-4ff1-b936-6ad8c0f547ca)
